aci-builder/build.sh

```diff
@@ -16,26 +16,53 @@ mkdir -p ${rootfs}/dgr ${rootfs}/usr/bin
 GOOS=linux GOARCH=amd64 go build --ldflags '-s -w -extldflags "-static"' -o ${rootfs}/dgr/builder/stage1/run ${dir}/bin-run
 upx ${rootfs}/dgr/builder/stage1/run
 
-sudo tar xf ${dir}/rootfs.tar.xz -C ${rootfs}/dgr/
+: ${tar:="$(realpath "${dir}/files/dgr/usr/bin/tar")"}
+if [[ ! -x "${tar}" ]] || ! "${tar}" --help | grep -q -F sort; then
```

Comment:
I don't like that, we should update buildroot instead since 1.29 is in here.
But since it's up to me to do it, I can accept it for now.